<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 255, 0);</string>
  </property>
  <widget class="QTableView" name="tab_emp">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>701</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_mod">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Modificar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_atr">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Atras</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
